test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -138,6 +152,10 @@ public void SetupPythonTests()
             var assemblyPath = Assembly.GetExecutingAssembly().Location;
             var moduleRootFolder = Path.GetDirectoryName(assemblyPath);
 
+            Assembly.LoadFrom(Path.Combine(moduleRootFolder, "DSCPython.dll"));
+          //  Assembly.LoadFrom(Path.Combine(moduleRootFolder, "Python.Included.dll"));
```

Comment:
@reddyashish whats this commented out code?

Reply:
Ah, yes forgot to remove it. Was testing if we needed we these binaries for the tests.